README.md

```diff
@@ -207,7 +216,7 @@ The configuration properties needed to run the tests are:
 Pytest markers select what pytests to run in `model-config-tests`. Current markers include:
 
 - `checksum`: Historical reproducibility test that runs a model and compares checksums with a stored previous result.
-- `checksum_slow`: Restart reproducibility tests. These include model determinism, by repeating running a configuration from initial conditions to check it reproduces the same output, and checking reproducibility by confirming two short consecutive model runs give the same result as a longer single model run.
+- `checksum_slow`: Restart reproducibility tests. These include model determinism, by repeating running a configuration from initial conditions to check it reproduces the same output (`Determinism rest`--see above `Definitions for reproducibility`), and checking reproducibility by confirming two short consecutive model runs give the same result as a longer single model run (`Restart reproducibility`--see above `Definitions for reproducibility`)).
```

Comment:
This is not actually correct anymore. Apologies, I should have updated as part of https://github.com/ACCESS-NRI/model-config-tests/pull/137. Do you mind if I push an update to this branch?

Reply:
Done in [a8b5423](https://github.com/ACCESS-NRI/model-config-tests/pull/145/commits/a8b5423132f19b6073a87af09207b481d1566b3b)